memcached.c

```diff
@@ -5557,40 +5558,31 @@ static void drive_machine(conn *c) {
                 void *ssl_v = NULL;
 #ifdef TLS
                 SSL *ssl = NULL;
-                if (IS_TCP(c->transport) && settings.ssl_enabled) {
-                    // Setup the SSL object at each connection level when it's enabled
-                    // at the correct port if ssl_port is specified.
-                    int port = 0;
-                    if (settings.ssl_port > 0) {
-                        struct sockaddr_in peer_addr;
-                        socklen_t peer_addr_size = sizeof(peer_addr);
-                        getsockname(sfd, (struct sockaddr *)&peer_addr, &peer_addr_size);
-                        port = ntohs(peer_addr.sin_port);
+                if (IS_TCP(c->transport) &&
+                    settings.ssl_enabled &&
+                    c->ssl_enabled) {
+                   if (settings.ssl_ctx == NULL) {
+                        fprintf(stderr, "SSL context is not initialized\n");
```

Comment:
needs an "if verbose" check, along with the other errors below.

Reply:
+1